### PR TITLE
WIP: Ovirt port template to workflow 

### PR DESCRIPTION
**Pull request by eslutsky**
continue work from https://github.com/openshift/release/pull/13943

### CODE REVIEW

File: ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh [hunk 1]
```diff
@@ -58,6 +58,7 @@ aws)
 azure4) export TEST_PROVIDER=azure;;
 vsphere) export TEST_PROVIDER=vsphere;;
 openstack) export TEST_PROVIDER='{"type":"openstack"}';;
+ovirt) export TEST_PROVIDER='{"type":"ovirt"}';;
```
Comment (petr-muller) [inline ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh:61]: @eslutsky This is a culprit of why your PR is triggering so many useless rehearsals; this step is used in a million worfklows and tirggers their rehearsals, which sometimes (it's a little non-deterministic) causes the ovirt jobs to not be rehearsed.

I'd suggest leaving this change out entirely while you are working on the steps, and only after you actually successfuly hit the "Unsupported cluster type" message in an ovirt rehearsal, put it back it. That way you won't be changing this hotspot step and you will only trigger rehearsals of the ovirt job.

Comment (eslutsky) [reply ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh]: ack , removing it for now

**Pull request closed without merging**